deno.jsonc

```diff
@@ -7,7 +7,7 @@
     "semiColons": false
   },
   "tasks": {
-    "test": "deno test --allow-net --allow-read --coverage=coverage",
```

Comment:
deno task test is used in GH actions, please keep it

Reply:
my bad.